utils/storage.ts

```diff
@@ -5,6 +5,19 @@ import { none, Option, some } from "fp-ts/lib/Option";
 
 import { ITuple2, Tuple2 } from "italia-ts-commons/lib/tuples";
 
+/**
+ * A promisified version of TableService.createTableIfNotExists
+ */
+export const createTableIfNotExists = (
+  tableService: TableService,
+  table: string
+): Promise<Either<Error, TableService.TableResult>> =>
+  new Promise(resolve =>
+    tableService.createTableIfNotExists(table, (error, result, response) =>
+      resolve(response.isSuccessful ? right(result) : left(error))
+    )
+  );
```

Comment:
I did this just to keep the functions in this class similar in concept. Anyway it could be a nice way to show to new team members a way to avoid promises. I "taskify" it.

Reply:
Done.